src/commands/new.js

```diff
@@ -8,6 +8,20 @@ const questions = [
     name: 'path',
     message: 'Project folder name',
     default: 'my-api'
+  },
+  {
+    type: 'input',
+    name: 'region',
+    message: 'Region for project',
+    default: 'us-east-1',
+    config: true
+  },
+  {
+    type: 'input',
+    name: 'rolename',
+    message: 'Enter the name of the IAM role which you wish to use, if it is not found it will be created',
+    default: 'shepRole',
```

Comment:
What is the default role the lambda does if you create your first lambda func via the UI? What if we made that the default?